src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,14 +1,30 @@
 package core.basesyntax.service;
 
-import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
-    private final StorageDao storageDao = new StorageDaoImpl();
-
     @Override
     public User register(User user) {
+        if (user == null) {
+            throw new RuntimeException("User object can't be null");
```

Comment:
we need to write our own exception
```suggestion
            "you own exception"("User object can't be null");
```
do this with all condition